spec/index.bs

```diff
@@ -1779,6 +1802,9 @@ throws an "`EvalError`" if not:
 
 1.  <ins>If |sourceString| is not equal to |source|, throw an {{EvalError}}.</ins>
 
+1.  <ins>Let |requireTrustedTypes| be the result of executing [$Does sink type require trusted types?$] algorithm,
```

Comment:
As currently written this doesn't deal with the case where script-src is in enforcement mode but requre-trusted-types-for isn't.